msgcntr/messageforums-app/src/webapp/js/forumTopicThreadsSorter.js

```diff
@@ -367,16 +366,18 @@ jQuery.fn.threadsSorter = function() {
 			if(flip %2 == 0){
```

Comment:
These two blocks of code could be put in a single function that takes a boolean.

https://gist.github.com/adrianfish/3842cb1205950f3386a77fad93c1b949

Reply:
Thanks for the suggestion, works correctly